src/beamlime/core/orchestrating_processor.py

```diff
@@ -135,12 +140,12 @@ def process(self) -> None:
         workflow_data = self._message_preprocessor.preprocess_messages(message_batch)
 
         # Handle data messages with the workflow manager, accumulating data as needed.
-        job_statuses = self._job_manager.push_data(workflow_data)
+        job_errors = self._job_manager.push_data(workflow_data)
 
         # Log any errors from data processing
-        for status in job_statuses:
-            if status.has_error:
-                self._logger.error(self._job_manager.format_job_error(status))
+        for error in job_errors:
+            if error.has_error:
```

Comment:
Maybe related to the comment above about confusion as to what is warning and what is error:
I find it strange that an `error` potentially does not have an error...